test/tools/spec-runner/matcher.js

```diff
@@ -34,6 +34,16 @@ function generateMatchAndDiffSpecialCase(key, expectedObj, actualObj, metadata)
       };
     }
 
+    if (actualObj._bsontype === 'Code' && key === 'scope') {
```

Comment:
why has that not come up before?

Reply:
See comment about `type=*` tests. Code has a `scope: Document | null` property now instead of `scope?: Document`. The prop always exists, its explicitly set to null when there is no scope.